src/ucs/sys/sys.c

```diff
@@ -407,6 +427,11 @@ size_t ucs_get_phys_mem_size()
 
     if (phys_pages == 0) {
         phys_pages = ucs_sysconf(_SC_PHYS_PAGES);
+        if (phys_pages < 0) {
+            phys_pages = SIZE_MAX;
```

Comment:
would cause overflow. need to *return" SIZE_MAX

Reply:
ouch